committree.go

```diff
@@ -10,7 +10,11 @@ import (
 )
 
 func (c *Client) GetAuthor() string {
-	configFile, err := os.Open(os.Getenv("HOME") + "/.gitconfig")
+	home := os.Getenv("HOME")
+	if home == "" {
+		home = os.Getenv("home") // On some OSes, it is home
+	}
```

Comment:
It would probably be better to add a getHomeDir() function which uses build tags to only do the correct thing for the OS that the binary is being built for, but that can be fixed later. (or better yet, use the standard `os.user` package which already does the build tag magic..)